### PR TITLE
feature/ST-242-improve-apple-pay-error-reasons

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

File: src/core/integrations/ApplePay.ts [hunk 1]
```diff
@@ -292,146 +319,208 @@ export class ApplePay {
...
       });
     }
   }
+
+  /**
+   * Handles errors during ApplePay process.
+   * @param errorObject
+   * @private
+   */
+  private _handleApplePayError(errorObject: any) {
```
Comment (james-simpson-ST) [inline src/core/integrations/ApplePay.ts]: Have we unit tested this?

Comment (pgs-gmartowski) [reply src/core/integrations/ApplePay.ts]: oops, sorry, it's not covered :/

Comment (pgs-gmartowski) [reply src/core/integrations/ApplePay.ts]: Done.
I'm not sure if my implementation is ok, I couldn't invoke such errors in ApplePay :/

File: webpack.common.js [hunk 2]
```diff
@@ -142,7 +142,7 @@ module.exports = {
           {
             loader: 'tslint-loader',
             options: {
-              emitErrors: true
+              emitErrors: false
```
Comment (james-simpson-ST) [inline webpack.common.js]: Need to revert this

Comment (pgs-gmartowski) [reply webpack.common.js]: done

**Pull request merged**